app/views/support_interface/application_forms/application_choices/reinstate_offer.html.erb

```diff
@@ -0,0 +1,40 @@
+<% content_for :browser_title, title_with_error_prefix('Reinstate offer', @declined_course_choice.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(application_form_id: params[:application_form_id]), 'Back to application') %>
+
+<%= form_with model: @declined_course_choice, url: support_interface_application_form_reinstate_offer_path(application_form_id: params[:application_form_id], application_choice_id: params[:application_choice_id]) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+    <%= f.govuk_error_summary %>
+      <h1 class="govuk-heading-l">
+        Are you sure you want to reinstate the offer for this course choice?
```

Comment:
@paulrobertlloyd is it worth also mentioning the actual course choice they are about to update somewhere on this page as well... or in this heading?